site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -1,51 +1,60 @@
 ---
 layout: 'layouts/doc-post.njk'
-
-# The page title. This appears at the top of the doc and as the page name
-# in Google Search.
 title: Welcome to Manifest V3
-
-# This appears below the title and is an optional teaser
 subhead: 'A step in the direction of security, privacy, and performance.'
-
-# This appears in the ToC of the project landing page at
-# /docs/[project-name]/. It also appears in the <meta description> used in 
-# Google Search.
 description: 'An introduction to Manifest V3'
-
-# The publish date
 date: 2020-11-09
+updated: 2022-09-02
+---
 
-# An optional updated date
-# updated: 2020-10-16
+This site introduces Manifest V3, short for Manifest Version 3, which is the latest iteration of the Chrome extension platform. It shares the background and reasons for introducing Manifest V3 and the vision for the platform's future, along with resources on how to migrate.
 
-# A list of authors. These usernames correspond to the keys in the
-# _data/authorsData.json file.
+## What is a Manifest?
```

Comment:
```suggestion
## What is a manifest?
```

Reply:
This is a common noun not a proper noun. The way to remember which is which is that common nouns are classes and proper nouns are instances.